backend/src/database/populationUtils/userPopulation.ts

```diff
@@ -0,0 +1,17 @@
+import { User, UserType } from '../models/user';
+import { addUser } from '../collectionsUtils/userUtils';
+import faker from 'faker';
+
+export const populateUsersCollection = async (n = 10, role = "student") => {
+    for (let i = 1; i <= n; i++) {
+      addUser({
+        name: faker.name.firstName(),
+        surname: faker.name.lastName(),
+        email: faker.internet.exampleEmail(),
+        password: faker.internet.password(10),
```

Comment:
użytkownicy są bezużyteczni jeśli ich hasła w bazie sa zapisane w plaintextem bo przy logowaniu jest bcrypt

Reply:
No to napisz "dodaj proszę bcrypta w hasłach użytkowników, bo tak jest we froncie"
Zmiana nie będzie problemem